core/kernel/persistence/interface.PropertyInterface.php

```diff
@@ -83,7 +83,12 @@ public function delete(core_kernel_classes_Resource $resource, $deleteReference
      * @return core_kernel_classes_Class
      */
     public function setRange(core_kernel_classes_Resource $resource, core_kernel_classes_Class $class);
-
+
+    public function setDependsOnProperty(
+        core_kernel_classes_Resource $resource,
+        core_kernel_classes_Property $property
+    ): int;
```

Comment:
https://github.com/oat-sa/generis/pull/911/files#r694105283

I think a setter should not have number of affected rows in its interface. This is not a setter anymore.